src/index.html

```diff
@@ -398,7 +398,7 @@ <h2 class="pageItemFooter" style="display: block; font-size: 1rem;">
                         }'>
                         <img width=30px src="" data-bind="attr: { src: DayCycle.image() }"/>
                     </button>
-                    <button class='btn btn-sm' style="position: absolute; right: 0px; top: 0px; width: auto; height: 41px;"
+                    <button class='btn btn-sm' style="position: absolute; right: 48px; top: 0px; width: auto; height: 41px;"
```

Comment:
As I said on Discord, I have concerns that having two images with different moons at night will confuse folks. is there any way to synchronize them? Or add somewhere an explainer NPC? Or something in the FAQ?

Reply:
Yeah, that's probably not great. I'll can experiment with it maybe being used as the Night graphic, during the night, if that's the preferred way to handle it. Adding something to the FAQ or adding an NPC is also possible, though the wording could be hard with that.